apps/web/vite.config.ts

```diff
@@ -224,6 +226,6 @@ export default defineConfig(({ command }) => ({
         ],
       },
     }),
-    compression(),
+    compression({ exclude: /index\.html/i }),
```

Comment:
Could you please add a comment to remind us why it is excluded?

Reply:
Good idea, added in [document reason for not compressing index.html](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11351/commits/3cfdab010ccc95e3c9dec6743c858423bb77e9d7)